OSOE-593: Update Node.js Extensions to latest release incl. extra logging

0liver opened the pull request:
[OSOE-593](https://lombiq.atlassian.net/browse/OSOE-593)

[OSOE-593]: https://lombiq.atlassian.net/browse/OSOE-593?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Commit:
Update NE to latest

Commit:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-593

Commit:
Update NE to minor fix

Commit:
Write step summary with tool versions

Commit:
Fix build: escape & in XML command

Commit:
Add missing step title in LGHA

Commit:
Ignore unneeded jobs

Commit:
Get job summary printed

Commit:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-593

# Conflicts:
#	.github/workflows/build-and-test.yml

Commit:
Run 5 windows builds

Commit:
Increase timeout for default runners

Commit:
Print job summary with markdown - again

Commit:
Run job summary in default shell

Commit:
Add missing shell

Commit:
Switching to pwsh

Commit:
Using a table

Commit:
Restore build, run summary always

Commit:
Only run windows build again

Commit:
Revert build workflow

Commit:
Empty trigger commit

dministro committed:
Updating submodule:
 - Lombiq.GitHub.Actions: Don't print the config summary in case...

Commit:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-593

# Conflicts:
#	tools/Lombiq.GitHub.Actions

Commit:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-593

# Conflicts:
#	tools/Lombiq.GitHub.Actions

Commit:
Shared and Exclusive lock for NE

Commit:
Fix compilation errors

Commit:
Fix analyzer warnings, improve code

Commit:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-593

# Conflicts:
#	tools/Lombiq.GitHub.Actions

Commit:
Merge branch 'dev' into issue/OSOE-593

# Conflicts:
#	tools/Lombiq.GitHub.Actions

Commit:
Actually use the SharedMutex for "pnpm build"

Commit:
Fix analyzer warnings

Commit:
Final changes

dministro commented on src/Utilities/Lombiq.Npm.Targets:
I can't find the PR for this change.

0liver replied:
My apologies. Here it is: https://github.com/Lombiq/NPM-Targets/pull/35.

Commit:
Move NE test project into new Utilities folder underneath test

Commit:
Heavy refactoring of NE

Commit:
Fix new analyzer warning

dministro committed:
Targeting dev in workflow
Updating submodules:
 - Lombiq.GitHub.Actions: fast-forward
 - Lombiq.NodeJs.Extensions: fast-forward
 - Lombiq.Npm.Targets: fast-forward
